TODO.md

```diff
@@ -66,10 +66,10 @@ This phase will include all actions and activities that enable you to conduct a
 ⬜ Action: Authenticate Passcode Registered User on Program using Form Factor-Card
```

Comment:
Please add a green check mark here to show the whole action is completed

Reply:
Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)